MAINTAINERS.md

```diff
@@ -10,7 +10,9 @@ for these platforms.
 | Allwinner A80 Board			|`Sun Yangbang <sunny@allwinnertech.com>`|
 | ARM Juno Board			|`Linaro <op-tee@linaro.org>`|
 | Atmel ATSAMA5D2-XULT Board		|`Akshay Bhat <akshay.bhat@timesys.com>`|
-| FSL ls1021a				|`Sumit Garg <sumit.garg@freescale.com>`|
+| FSL ls1021a				|`Sumit Garg <sumit.garg@nxp.com>`|
+| NXP ls1043ardb			|`Sumit Garg <sumit.garg@nxp.com>`|
+| NXP ls1046ardb			|`Sumit Garg <sumit.garg@nxp.com>`|
```

Comment:
Minor:
I'm a bit confused about naming here. You replaced FSL with Layerscape in `.travis.yml`, and here you're introducing a mix of `FSL` and `NXP`. Should that be `NXP Layerscape ls1021a`, `NXP Layerscape ls1043ardb` etc. maybe?

Reply:
Sure I will fix it.